cli/azd/pkg/tools/docker/docker.go

```diff
@@ -41,11 +41,14 @@ type docker struct {
 }
 
 func (d *docker) Login(ctx context.Context, loginServer string, username string, password string) error {
-	_, err := d.executeCommand(ctx, ".", "login",
+	runArgs := exec.NewRunArgs(
+		"docker", "login",
 		"--username", username,
-		"--password", password,
-		loginServer)
+		"--password-stdin",
+		loginServer,
+	).WithCwd(".").WithStdIn(strings.NewReader(password))
```

Comment:
Curious.. Does this fail without `WithCwd` set? I'd expect the default behavior is that `CWD` is already set to an existing directory.